dashboard/app/dsl/script_dsl.rb

```diff
@@ -113,8 +130,16 @@ def variants
   end
 
   def endvariants
+    @current_scriptlevel[:properties][:prompt] = @prompt if @prompt
     @scriptlevels << @current_scriptlevel
+
+    unused_prop = (!@active && @active.to_s) || @buttontext || @imageurl || @description
```

Comment:
This is a weird statement, since `@active` defaults to true. I want `unused_prop` to contain the unused property value (e.g. "false" if `@active` is false) to make the exception message accurate.

Replaced with a ternary statement that's hopefully clearer.